### DISCUSSION TITLE
Pyinstaller --nonconsole flashing console windows

### DISCUSSION THREAD

**Discussion by user123-source** [PaddlePaddle/PaddleOCR#12807] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows 10
- 版本号/Paddle： 2.3  PaddleOCR： 2.6

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):yes

Have a fairly simple GUI tkinter app and is working great. I'm using Windows 10.
When compiling it using pyinstaller with -w (or --noconsole or --windowed) and run it, I can see a console window for a millisecond and then it shutdown.

if i remove the PaddleOCR import the console windows do not flash anymore.

please help

**Comment by user123-source** [comment]
How would this affect the library?

```
import subprocess
import sys

if sys.platform == "win32":
    CREATE_NO_WINDOW = 0x08000000  # this flag will prevent the window from showing
    subprocess.call('your_command_here', creationflags=CREATE_NO_WINDOW)
else:
    subprocess.call('your_command_here')

```

**Comment by breath8** [comment]
This issue might be caused by the library paddlepaddle. Please consider the following: https://github.com/PaddlePaddle/Paddle/pull/64666/commits/7a4f1b51c1cfffaeba95171b6bfe9864123f18a8
